Source/Noesis.Javascript/JavascriptExternal.cpp

```diff
@@ -50,15 +50,22 @@ using namespace std;
 
 JavascriptExternal::JavascriptExternal(System::Object^ iObject)
 {
-	mObjectHandle = System::Runtime::InteropServices::GCHandle::Alloc(iObject);
+	// .NET 8: Store GCHandle as IntPtr for safe native storage

```

Comment:
Why the change to handles at all? What benefit do they have? (see comment on PR for more details)